tests/integration/targets/aci_tenant_span_src_group_src/tasks/main.yml

```
@@ -0,0 +1,191 @@
+# Test code for the ACI modules
+# Copyright: (c) 2022, Akini Ross (@akinross)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
+    msg: 'Please define the following variables: aci_hostname, aci_username and aci_password.'
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+- name: Set vars
+  set_fact: 
+   aci_info: &aci_info
+    host: "{{ aci_hostname }}"
+    username: "{{ aci_username }}"
+    password: "{{ aci_password }}"
+    validate_certs: '{{ aci_validate_certs | default(false) }}'
+    use_ssl: '{{ aci_use_ssl | default(true) }}'
+    use_proxy: '{{ aci_use_proxy | default(true) }}'
+    output_level: '{{ aci_output_level | default("info") }}'
+
+# CLEAN ENVIRONMENT
+- name: Remove the ansible_tenant
+  cisco.aci.aci_tenant:
+    <<: *aci_info 
+    tenant: ansible_tenant
+    state: absent
+
+- name: Add a new tenant
+  cisco.aci.aci_tenant: &aci_tenant
+    <<: *aci_info
+    tenant: ansible_tenant
+    state: present
+
+- name: Add new bd
+  cisco.aci.aci_bd: &aci_bd
+    <<: *aci_tenant
+    bd: anstest
+  register: bd_present
+
+- name: Add a new source ap
+  cisco.aci.aci_ap:
+    <<: *aci_tenant 
+    ap: ansible_source_ap
+
+- name: Add a new dest ap
+  cisco.aci.aci_ap:
+    <<: *aci_tenant 
+    ap: ansible_dest_ap
+
+- name: Add a new dest epg
+  cisco.aci.aci_epg:
+    <<: *aci_bd 
+    ap: ansible_dest_ap
+    epg: ansible_dest_epg
+
+- name: Add a new source epg
+  cisco.aci.aci_epg:
+    <<: *aci_bd
+    ap: ansible_dest_ap
+    epg: ansible_source_epg
+
+- name: Add span dest group
+  cisco.aci.aci_tenant_span_dst_group:
+    <<: *aci_info
+    destination_group: ansible_dest_group
+    destination_ip: 10.0.0.1
+    source_ip: 10.0.2.1
+    tenant: ansible_tenant
+    ttl: 2
+    mtu: 1500
+    flow_id: 1
+    dscp: "CS1"
+    destination_epg:
+      tenant: ansible_tenant
+      ap: ansible_dest_ap
+      epg: ansible_dest_epg
+    state: present
+
+- name: Create a source group
+  cisco.aci.aci_tenant_span_src_group:
+    <<: *aci_tenant
+    name: ansible_span
+    description: ansible test description
+    dst_group: ansible_dest_group
+  register: nm_create_with_dest
+
+- name: Create a source (check mode)
+  cisco.aci.aci_tenant_span_src_group_src: &aci_src
+    <<: *aci_tenant
+    name: ansible_source
+    description: ansible test description
+    direction: incoming
+    src_ap: ansible_source_ap
+    src_epg: ansible_source_epg
+    src_group: ansible_span
+  check_mode: yes
+  register: cm_create_with_epg
+
+- name: Create a source
```

```suggestion
- name: Create ansible_source SPAN Source
```

Please add the detailed task name for other tasks.